src/activities/GradeCandidateCollectionEntity.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
Old habits die hard - ES6 modules are all strict mode, so don't need this anymore - but I find myself doing it all the time anyway! 😄